### PR TITLE
:tractor: Refactors relay + dj-database-url configuration

### LINKED ISSUE #55: Remove `dj-database-url` from project dependencies
It's only used in the relay service Docker container, so it should be removed from the project's dependencies in `pyproject.toml` and just installed directly in the service's `Dockerfile`.

### PR DESCRIPTION
**Pull request by jefftriplett**
Fixes #55 

This needs more opinions.

### CODE REVIEW

**Review by joshuadavidthomas — COMMENTED** [review]
It's a small nit, but I would go for the oneliner in this case:

```
python -m pip install '.[psycopg,relay]'
```

File: Dockerfile [hunk 1]
```diff
@@ -11,7 +11,8 @@ FROM base as py
 COPY src /app/src
 COPY pyproject.toml README.md /app
 RUN python -m pip install --upgrade pip \
-  && python -m pip install '.[psycopg]'
+  && python -m pip install '.[psycopg]' \
+  && python -m pip install '.[relay]'
```
Comment (jefftriplett) [inline Dockerfile:15]:
```suggestion
  && python -m pip install '.[psycopg,relay]'
  ```

**Comment by jefftriplett** [comment]
> It's a small nit, but I would go for the oneliner in this case:
> 
> ```
> python -m pip install '.[psycopg,relay]'
> ```

good nitpick. updated. 👍

**Review by joshuadavidthomas — APPROVED** [review]

**Pull request merged**